### PR TITLE
Internal 126 downgrade grpc

### PR DESCRIPTION
**Pull request by andrea-grapl**
### What changes does this PR make to Grapl? Why?

Downgrade grpc to see if updated version introduced bug in retrieving graph

### How were these changes tested?

Locally + Manually.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1 +1,6 @@
-{}
+{
+  "dependencies": {
+    "dgraph-js": "^20.3.1",
```
Comment (jgrillo-grapl) [inline package.json:3]: We run DGraph version 20.07.1. It's not clear whether the JS client supports this version from the table here: https://github.com/dgraph-io/dgraph-js#supported-versions

Looking at the releases page, the most recent release is 20.03.0: https://github.com/dgraph-io/dgraph-js/releases

Did you find version 20.3.1 published somewhere? It doesn't seem like it should exist...

**Review by jgrillo-grapl — CHANGES_REQUESTED** [review]
Seems like something wrong with the dgraph-js version

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "dgraph-js": "^20.3.1",
+    "dgraph-js": "^20.3.0",
```
Comment (jgrillo-grapl) [inline package.json:3]: This should be `20.3.0` not `^20.3.0`

**Review by jgrillo-grapl — CHANGES_REQUESTED** [review]

**Comment by andrea-grapl** [comment]
This was not causing the issue with DGraph. Closing issue.

**Pull request closed without merging**